.circleci/config.yml

```diff
@@ -106,14 +118,16 @@ commands:
         default: false
     steps:
       - checkout
+      - swissknife/trigger-pipeline:
+          install-skip-trigger: true
       - run:
           name: Tests
           command: |
             mkdir ./artifacts
             export COMPOSER_MEMORY_LIMIT=3G
             export TEST_ENV=<< parameters.php_version >>
             export MAGENTO_VERSION=<< parameters.m2_version >>
```

Comment:
these are unused here?

Reply:
Are you sure? These are not part of my changes, should I remove them?

Reply:
i mean they are unused right? it doesn't matter if it isn't a part of your change, unless they are being used lets remove them

Reply:
I tried removing them and it broke the build. I think these are being used